docs/LibraryEvolution.rst

```diff
@@ -700,8 +674,8 @@ still be modified in limited ways:
 - A versioned ``internal`` property may be made ``public`` (without changing
   its version).
 
-An initializer of a fixed-contents struct may be declared ``@inlineable`` even
-if it does not delegate to another initializer, as long as the ``@inlineable``
+An initializer of a fixed-contents struct may be declared ``@inlinable`` even
+if it does not delegate to another initializer, as long as the ``@inlinable``
```

Comment:
If the struct is not fixed-contents the initializer doesn't have to delegate to another initializer -- it can also assign to `self`.

Reply:
Ah, right.